content/zh/docs/concepts/security/pod-security-standards.md

```diff
@@ -0,0 +1,519 @@
+-->
+
+<!-- overview -->
+
+<!--
+Security settings for Pods are typically applied by using [security
+contexts](/docs/tasks/configure-pod-container/security-context/). Security Contexts allow for the
+definition of privilege and access controls on a per-Pod basis.
+
+The enforcement and policy-based definition of cluster requirements of security contexts has
+previously been achieved using [Pod Security Policy](/docs/concepts/policy/pod-security-policy/). A
+_Pod Security Policy_ is a cluster-level resource that controls security sensitive aspects of the
+Pod specification.
+
+However, numerous means of policy enforcement have arisen that augment or replace the use of
+PodSecurityPolicy. The intent of this page is to detail recommended Pod security profiles, decoupled
+from any specific instantiation.
+-->
+Pod 的安全性配置一般通过使用
+[安全性上下文（Security Context）](/zh/docs/tasks/configure-pod-container/security-context/)
+来保证。安全性上下文允许用户逐个 Pod 地定义特权级及访问控制。
+
+以前，对集群的安全性上下文的需求的实施及其基于策略的定义都通过使用
+[Pod 安全性策略](/zh/docs/concepts/policy/pod-security-policy/)来实现。
+_Pod 安全性策略（Pod Security Policy）_ 是一种集群层面的资源，控制 Pod 规约中
+安全性敏感的部分。
+
+不过，新的策略实施方式不断涌现，或增强或替换 PodSecurityPolicy 的使用。
+本页的目的是详细介绍建议实施的 Pod 安全框架；这些内容与具体的实现无关。
+
+<!-- body -->
+
+<!--
+## Policy Types
+
+There is an immediate need for base policy definitions to broadly cover the security spectrum. These
+should range from highly restricted to highly flexible:
+
+- **_Privileged_** - Unrestricted policy, providing the widest possible level of permissions. This
+  policy allows for known privilege escalations.
+- **_Baseline/Default_** - Minimally restrictive policy while preventing known privilege
+  escalations. Allows the default (minimally specified) Pod configuration.
+- **_Restricted_** - Heavily restricted policy, following current Pod hardening best practices.
+-->
+## 策略类型   {#policy-types}
+
+在进一步讨论整个策略谱系之前，有必要对基本的策略下个定义。
+策略可以是很严格的也可以是很宽松的：
+
+- **_Privileged_** - 不受限制的策略，提供最大可能范围的权限许可。这些策略
+  允许已知的特权提升。
+- **_Baseline/Default_** - 限制性最弱的策略，禁止已知的策略提升。
+  允许使用默认的（规定最少）Pod 配置。
+- **_Restricted_** - 限制性非常强的策略，遵循当前的保护 Pod 的最佳实践。
+
+<!--
+## Policies
+
+### Privileged
+-->
+## 策略    {#policies}
+
+### Privileged
+
+<!--
+The Privileged policy is purposely-open, and entirely unrestricted. This type of policy is typically
+aimed at system- and infrastructure-level workloads managed by privileged, trusted users.
+
+The privileged policy is defined by an absence of restrictions. For allow-by-default enforcement
+mechanisms (such as gatekeeper), the privileged profile may be an absence of applied constraints
+rather than an instantiated policy. In contrast, for a deny-by-default mechanism (such as Pod
+Security Policy) the privileged policy should enable all controls (disable all restrictions).
+-->
+Privileged 策略是有目的地开放且完全无限制的策略。此类策略通常针对由
+特权较高、受信任的用户所管理的系统级或基础设施级负载。
+
+Privileged 策略定义中限制较少。对于默认允许（Allow-by-default）实施机制，
+Privileged 框架可能意味着不应用任何约束而不是实施某策略实例。
+与此不同，对于默认拒绝（Deny-by-default）实施机制（如 Pod 安全策略）而言，
```

Comment:
```suggestion
与此不同，对于默认拒绝（Deny-by-default）实施机制（如 PodSecurityPolicy）而言，
```
?

Reply:
The English text was using `Pod Security Policy` rather than `PodSecurityPolicy`, which IMHO is the right way to put it. `PodSecurityPolicy` is the resource to implement the mechanism `Pod Security Policy`. When the words are split, we translate accordingly, otherwise we keep the resource name spelling in English.